src/services/WolfDen.API/Controllers/Department/Department.cs

```diff
@@ -12,10 +14,16 @@ public class Department(IMediator mediator) : ControllerBase
         private readonly IMediator _mediator=mediator;
 
         
-        [HttpPost]
+        [HttpPost("add-department")]
```

Comment:
Ideally this is not required, [POST] request in a given controller usually denotes the creation/ adding of the Entity related to the controller.
Is there any specific reason why `add-department` was added?

Reply:
no, corrected